guide/case-studies/blixt-wallet.md

```diff
@@ -0,0 +1,154 @@
+
+The [Bitcoin Design Community](https://bitcoindesign.slack.com/join/shared_invite/zt-10sxfovaq-isViijl4RThKRs_TsAQnuA#/shared-invite/email) ran a design sprint on an open-source lightning wallet and during this sprint we collaboratively focused on re-designing one aspect of the user flow, the **onboarding journey**. During this collaborative effort, [we got together](https://www.youtube.com/watch?v=nctibe_a4rM&list=PLpV0KfVOMojYywq1Blp3SGx62cSyoRfHt&index=3) and as a group of designers and researchers we aimed to improve the user experience by using good design principles incorporating as much feedback from users as possible. 
+
+<p class="h3">Blixt Wallet is a non-custodial open-source Bitcoin Lightning Wallet for Android and iOS and aims to serve users who are new to lightning, but are familiar with bitcoin and want to try out the lightning network for the first time.</p>
+
+The current user base of Blixt tends to be more advanced users and Blixt would like to move towards catering more to users who want to try out the lightning network as well. 
+
+What does Blixt do?
+
+- Onboard someone quickly over lightning so they can have a channel already provisioned to receive funds
+- Automatically open channels anytime the user has on-chain funds (Auto pilot)
+- Automatic channel creation (using [Dunder LSP](https://github.com/hsjoberg/dunder-lsp)) during initial deposit of funds
+- Support for custom LSP
+
+
+{% include picture.html
+   image = "/assets/images/guide/case-studies/blixt-wallet/blixt-users.jpg"
+   retina = "/assets/images/guide/case-studies/blixt-wallet/blixt-users@2x.jpg"
+   mobile = "/assets/images/guide/case-studies/blixt-wallet/blixt-users-mobile.jpg"
+   mobileRetina = "/assets/images/guide/case-studies/blixt-wallet/blixt-users.jpg"
+   alt-text = "Characters representing Blixt's users"
+   caption = "Illustrations: [OpenPeeps](https://www.openpeeps.com/) by Pablo Stanley"
+   width = 800
+   height = 450
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/case-studies/blixt-wallet/blixt-ux-process.jpg"
+   retina = "/assets/images/guide/case-studies/blixt-wallet/blixt-ux-process@2x.jpg"
+   mobile = "/assets/images/guide/case-studies/blixt-wallet/blixt-ux-process-mobile.jpg"
+   mobileRetina = "/assets/images/guide/case-studies/blixt-wallet/blixt-ux-process.jpg"
+   alt-text = "Diagram of stages in the Blixt UX process"
+   width = 800
+   height = 929
+%}
+
+Since Blixt is an open source wallet we collected user research simply by going on their Discord channel and collecting comments and questions from users.
+
+## We discovered that Blixt users were struggling to open lightning channels.
+
+We created a lot of different user flows to find a solution for the best method to open a channel for a user new to lightning, and eventually settled on this one. We decided on this user flow as it educates users first and then takes them to setting up the channel itself. 
```

Comment:
> eventually settled on this one

Is there a link or an image missing here, I wasn't able to understand what "this one" refers to?

Reply:
Correct, there is a userflow image missing here. We will add that one in.

Reply:
Oh yeah, thanks for pointing that out. @mouxdesign I couldn't find it in the Figma file.

Reply:
Moved the user flow image in the figma file, it would be the userflow one. Dropping an image of it below so you have an idea of what to look for in the figma file. 
![Group 2435 (1)](https://user-images.githubusercontent.com/90313394/202697099-d1f4590f-9deb-4c94-a338-12a62c48acb5.jpg)

This userflow would ideally go under that text "We created a lot of different user flows to find a solution for the best method to open a channel for a user new to lightning, and eventually settled on this one. We decided on this user flow as it educates users first and then takes them to setting up the channel itself. ".

Reply:
@sbddesign also gave you editing rights to the figma file.

Reply:
Done